source/_integrations/roku.markdown

```diff
@@ -138,3 +138,18 @@ action:
     media_content_type: channel
   service: media_player.play_media
 ```
+
+## Remote
+
+The `remote` platform allows you to send keypress commands to your device. Remote entities are automatically registered during integration setup.
+
+## Services
+
+### Service `roku.search`
+
+This service allows you to emulate opening the search screen and entering the search keyword.
+
+| Service data attribute | Optional | Description | Example |
+| ---------------------- | -------- | ----------- | ------- |
+| `entity_id` | no | The entities to search on. | media_player.roku
```

Comment:
Entity ID is optional, as an area could be targeted instead (when using the frontend automations editor). Support for areas is automatically available, as the parent PR uses `async_register_entity_service`

```suggestion
| `entity_id` | yes | The entities to search on. | media_player.roku
```